Source/Scene/Cesium3DTileset.js

```diff
@@ -317,14 +327,17 @@ define([
             return;
         }
 
-        var stats = tiles3D._statistics;
-        ++stats.numberOfPendingRequests;
-        addLoadProgressEvent(tiles3D);
-
         tile.requestContent();
-        var removeFunction = removeFromProcessingQueue(tiles3D, tile);
-        when(tile.processingPromise).then(addToProcessingQueue(tiles3D, tile)).otherwise(removeFunction);
-        when(tile.readyPromise).then(removeFunction).otherwise(removeFunction);
+
+        if (!tile.isContentUnloaded()) {
+            var stats = tiles3D._statistics;
+            ++stats.numberOfPendingRequests;
+            addLoadProgressEvent(tiles3D);
+
+            var removeFunction = removeFromProcessingQueue(tiles3D, tile);
+            when(tile.processingPromise).then(addToProcessingQueue(tiles3D, tile)).otherwise(removeFunction);
+            when(tile.readyPromise).then(removeFunction).otherwise(removeFunction);
+        }
     }
 
     function hasAvailableRequests(tiles3D) {
```

Comment:
I believe this line is wrong:
```javascript
return RequestScheduler.hasAvailableRequests(tiles3D._url)
```
This assumes that the tile's content has the same base path as tileset.json, right?  They could be on two separate servers.